aeon/classification/sklearn/_continuous_interval_tree.py

```diff
@@ -27,7 +27,7 @@ class _TreeNode:
     """ContinuousIntervalTree tree node."""
 
     def __init__(
-        self, random_state: Union[int, Type[np.random.RandomState], None] = None
+        self, random_state: Union[int, type[np.random.RandomState], None] = None
```

Comment:
and in this file as well:
```suggestion
        self, random_state: Union[int, np.random.RandomState, None] = None,
```